src/clj/game/cards/ice.clj

```diff
@@ -978,7 +978,8 @@
                   :async true
                   :effect (req (if (= target "Pay 3 [Credits]")
                                  (do (system-msg state :runner "pays 3 [Credits]")
-                                     (pay-sync state :runner card eid :credit 3))
+                                     (pay-sync state :runner card eid :credit 3)
+                                     (effect-completed state side eid))
```

Comment:
If this is needed, something more complex is broken, because the `eid` in the `pay-sync` call should be calling `effect-completed` for us.

Reply:
Oh duh, the arguments are backwards.